tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -271,233 +271,233 @@
     <string name="pref_dnscrypt_sources">Sources</string>
     <string name="pref_dnscrypt_sources_summ">Remote lists of available servers.</string>
-    <string name="pref_dnscrypt_refresh_delay" translatable="false">refresh_delay</string>
-    <string name="pref_dnscrypt_refresh_delay_summ">Sources list expiring after refresh_delay hours.</string>
+    <string name="pref_dnscrypt_refresh_delay" translatable="false">Refresh-Delay</string>
+    <string name="pref_dnscrypt_refresh_delay_summ">Lists of sources expire after an amount of hours set in \"Refresh Delay\".</string>
 
     <string name="pref_dnscrypt_relays_categ">Relays</string>
     <string name="pref_dnscrypt_relays">Sources</string>
-    <string name="pref_dnscrypt_relays_delay" translatable="false">refresh_delay</string>
-    <string name="pref_dnscrypt_relays_delay_summ">Relays list expiring after refresh_delay hours.</string>
+    <string name="pref_dnscrypt_relays_delay" translatable="false">Refresh-Delay</string>
+    <string name="pref_dnscrypt_relays_delay_summ">Lists of relays expire after an amount of hours set in \"Refresh-Delay\".</string>
     <string name="pref_dnscrypt_relays_title">DNSCrypt Relays</string>
 
 
-    <string name="pref_dnscrypt_not_exist">Preference does not exist in dnscrypt-proxy.toml!</string>
+    <string name="pref_dnscrypt_not_exist">The preference does not exist in dnscrypt-proxy.toml!</string>
 
     <string name="btn_dnscrypt_delete_log">Delete log</string>
     <string name="title_dnscrypt_query_log">DNSCrypt Query Log</string>
     <string name="title_dnscrypt_nx_log">DNSCrypt Suspicious Log</string>
-    <string name="dnscrypt_empty_log">Log is Empty</string>
+    <string name="dnscrypt_empty_log">The log is Empty</string>
     <string name="title_dnscrypt_forwarding_rules" translatable="false">DNSCrypt Forwarding rules</string>
     <string name="title_dnscrypt_cloaking_rules" translatable="false">DNSCrypt Cloaking rules</string>
     <string name="title_dnscrypt_blacklist" translatable="false">DNSCrypt Blacklist</string>
     <string name="title_dnscrypt_ip_blacklist" translatable="false">DNSCrypt IP Blacklist</string>
     <string name="title_dnscrypt_whitelist" translatable="false">DNSCrypt Whitelist</string>
 
     <string name="pref_dnscrypt_block_unqualified" translatable="false">block_unqualified</string>
-    <string name="pref_dnscrypt_block_unqualified_summ">Immediately respond to A and AAAA queries for host names without a domain name.</string>
+    <string name="pref_dnscrypt_block_unqualified_summ">Immediately respond to A and AAAA queries for hostnames without a domain name.</string>
     <string name="pref_dnscrypt_block_undelegated" translatable="false">block_undelegated</string>
     <string name="pref_dnscrypt_block_undelegated_summ">Immediately respond to queries for local zones instead of leaking them to upstream resolvers (always causing errors or timeouts).</string>
```

Comment:
```suggestion
    <string name="pref_dnscrypt_block_undelegated_summ">Immediately respond to queries for local zones instead of leaking them to upstream resolvers (causing errors or timeouts).</string>
```